cfme/physical/provider/__init__.py

```diff
@@ -157,5 +157,5 @@ class Add(CFMENavigateStep):
 
     def step(self):
         self.prerequisite_view.toolbar.configuration.item_select(
-            'Add a New Physical Provider'
+            'Add a New Physical Infrastructure Provider'
```

Comment:
This looks like it breaks it for Lenovo provider. Although I'm not sure how can this button be named differently for Lenovo and RedFish. Are we actually fixing a bug from Lenovo?

Reply:
Actually it doesn't break anything, since no Lenovo tests uses this step.

Reply:
via setup_provider fixture 
https://github.com/ManageIQ/integration_tests/blob/6c705763ba8cef4e5c4912074fe6dd0821e47bc1/cfme/tests/physical_infrastructure/ui/test_physical_server_list.py#L10-L13